mattermost/mattermost
# [GH-25484] Fix draft removal on post deletion

#### Summary
- This PR fixes the issue with the draft removal on deleting the post
- See this issue for more info: https://github.com/mattermost/mattermost/issues/25484

#### Steps to reproduce and test
- Create a post from account `A`
- Write something as a draft on that post from a different account `B`
- Delete a post from account `A`
- See Draft section on account `B`
- In the older version, the draft was still visible in the `Draft` section on the right side. But in this fix, it will be removed.

#### Ticket Link
Fixes https://github.com/mattermost/mattermost/issues/25484
https://mattermost.atlassian.net/browse/MM-55613

#### Release Note

```release-note
Fix Draft removal after deleting the post
```
